Looks like the build broke due to something not in my pull request.

pylint runtests: commands[0] | pylint homeassistant
************* Module homeassistant.helpers.device_registry
homeassistant/helpers/device_registry.py:45:4: R1705: Unnecessary "elif" after "return" (no-else-return)

--- a/homeassistant/components/sensor/srp_energy.py
+++ b/homeassistant/components/sensor/srp_energy.py
@@ -98,7 +98,6 @@ class SrpEnergy(Entity):
     @property
     def history(self):
         """Return the energy usage history of this entity, if any."""
-
         if self.data is None:
             return None
 
@@ -123,27 +122,9 @@ class SrpEnergy(Entity):
 
         return data
 
-    # @property
-    # def device_state_attributes(self):
-    #     """Return the history array."""
-
-    #     data = None
-
-    #     data = [{
-    #         ATTR_READING_TIME:
-    #             datetime.fromtimestamp(isodate),
-    #         ATTR_READING_USAGE:
-    #             kwh,
-    #         ATTR_READING_COST:
-    #             cost
-    #     } for date, hour, isodate, kwh, cost in self.data]
-
-    #     return data
-
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Get the latest data from SRP Energy."""
-
         start_date = datetime.now() + timedelta(days=-1)
         end_date = datetime.now()
 

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -1433,7 +1433,7 @@ spotipy-homeassistant==2.4.4.dev1
 sqlalchemy==1.2.13
 
 # homeassistant.components.sensor.srp_energy
-srpenergy=1.0.1
+srpenergy==1.0.1
 
 # homeassistant.components.sensor.starlingbank
 starlingbank==1.2

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -239,7 +239,7 @@ somecomfort==0.5.2
 sqlalchemy==1.2.13
 
 # homeassistant.components.sensor.srp_energy
-srpenergy=1.0.1
+srpenergy==1.0.1
 
 # homeassistant.components.statsd
 statsd==3.2.1

--- a/tests/components/sensor/test_srp_energy.py
+++ b/tests/components/sensor/test_srp_energy.py
@@ -1,16 +1,9 @@
 """The tests for the Srp Energy Platform."""
 import unittest
-from unittest.mock import MagicMock, patch
-from datetime import timedelta
-
-import requests_mock
-
-from srpenergy.client import SrpEnergyClient
-
-from homeassistant.components.sensor import srp_energy
+from unittest.mock import patch
 from homeassistant.setup import setup_component
 
-from tests.common import (load_fixture, get_test_home_assistant,
+from tests.common import (get_test_home_assistant,
                           MockDependency)
 
 VALID_CONFIG_MINIMAL = {

--- a/tests/components/sensor/test_srp_energy.py
+++ b/tests/components/sensor/test_srp_energy.py
@@ -77,4 +77,5 @@ class TestSrpEnergySetup(unittest.TestCase):
 
         state = self.hass.states.get('sensor.srp_energy')
 
-        assert state.daily_usage == 7.50
+        assert state.attributes is not None
+        assert state.attributes['daily_usage'] == '7.50'

